### PR TITLE
Add start streaming button

### LINKED ISSUE #3848: Expenditures: As a user I can start stream
[Provided by Adrian via Contractors project management platform]

## Description
User has to be able to start a stream by clicking on the button. He also should be able to display information about "Draft state" and "Paid to date".

## Acceptance Criteria

- an option to click the button and start stream
- an option to see a draft state (default "not saved")
- an option to see paid to date (default "not started")

## Design

- [Figma Link - Setup parameters with ‘Limit is reached’ Ends option selected](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5541%3A511994)

Until - popover 5

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds stages section in streaming payment type (UI only).
Screenshot 2022-09-14 at 10 10 13

**Changes** 🏗

* Changed ExpenditurePage file. The validationSchema and initialValues ​​objects have been moved from there to a separate file. 

## TODO

- In next PR add displaying modal after clicking 'Start Stream' button.

Resolves  #3848

### CODE REVIEW

**Review by Julianahlk — APPROVED** [review]
Nice one! My only question involves the front size, maybe @arrenv, would be able to clarify😊: Is the "Not started" text supposed to be bigger than the "Paid to date" text? From my perspective, there is a difference of about 1px which is making it look a bit inconsistent with the other text in this section.

Screen Shot 2022-09-15 at 3 44 54 PM

**Comment by arrenv** [comment]
> Nice one! My only question involves the front size, maybe @arrenv, would be able to clarify😊: Is the "Not started" text supposed to be bigger than the "Paid to date" text? From my perspective, there is a difference of about 1px which is making it look a bit inconsistent with the other text in this section.

The font sizes in the design are different and this seems to be on purposes as it is using defined font styles.

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStages.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: color-mod(var(--temp-grey-blue-7) alpha(85%));
+  letter-spacing: var(--spacing-medium);
+}
+
+.value {
+  font-size: var(--size-normal);
+  color: var(--dark);
+  letter-spacing: 0.1px;
```
Comment (arrenv) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStages.css:26]: Can we please make this font-weight bold

**Review by arrenv — APPROVED** [review]
Looks good @agnieszkajarosikloj , just one comment on the font.

**Comment by Julianahlk** [comment]
> 

Ok, thanks I wanted to confirm because it look like the font families are different as well.

**Review by chinins — APPROVED** [review]
Nice work!

**Pull request merged**